### ISSUE TITLE
Initial algorithm validation

### ISSUE THREAD

**Issue by kaulfusa** [NASA-IMPACT/veda-black-marble#4]
In order to do initial validation of the algorithm, the output should be compared against existing HD BM products in VEDA. 2 AOIs with 6 total dates are provided below

AOI for Hurricane Maria
"geometry":{"type":"Polygon","coordinates":[[[-67.27167653359618,17.912138994450856],[-65.57478762584185,17.912138994450856],[-65.57478762584185,18.51569455671654],[-67.27167653359618,18.51569455671654],[-67.27167653359618,17.912138994450856]]]}
and here is a list of dates
7/21/2017, 12:00:00 AM UTC
9/20/2017, 12:00:00 AM UTC
11/21/2017, 12:00:00 AM UTC
1/21/2018, 12:00:00 AM UTC

Hurricane Ida. Here's the AOI and dates
"geometry":{"type":"Polygon","coordinates":[[[-90.3037818244749,29.804659612978707],[-89.87578181971654,29.804659612978707],[-89.87578181971654,30.07177072705947],[-90.3037818244749,30.07177072705947],[-90.3037818244749,29.804659612978707]]]}
dates:
8/9/2021, 12:00:00 AM UTC
8/31/2021, 12:00:00 AM UTC

Outputs should include a visualization with our algorithm output, the data currently in VEDA, and a difference plot. These should be provided for review with major differences to be investigated further for potential remediation.

**Comment by kaulfusa** [comment]
@DImuthuUpe any update on the validation activity?

**Comment by DImuthuUpe** [comment]
@kaulfusa I am having issues in cropping images for the BB hurricane maria as the tiles belong to 2 UTM zones. When the algorithm tried to  crop it usin gdal translate, it did not crop to the expected bounding box. I am trying different approaches using QGIS. The other option is to convert all tiles into WGS84 before doing any operation

**Comment by DImuthuUpe** [comment]
This is the issue I am experiencing
https://www.youtube.com/watch?v=i6--NY6P56g

**Comment by kaulfusa** [comment]
This is with the Landsat data that spans multiple UTM zones? Another option would be to convert 1 UTM projection to another. 

One reason for not converting prior to operations was because of the quality loss. Can you make the bbox smaller to not span multiple zones so we can at least get a baseline on quality for multiple cases then we can take another look at projections and mosiacing.

**Comment by DImuthuUpe** [comment]
Sure. I will make the bounding box smaller and run. In the mean time, I managed to generate images using a global WGS84 conversion. Here are the outputs for different dates of the timeline 

https://drive.google.com/file/d/1YaU5rKcDJ9eron0R2XfSF0puTbYsMq3s/view?usp=sharing

**Comment by DImuthuUpe** [comment]
@kaulfusa I converted coordinates into UTM zone 20 projection grid and generated the output in the desired manner.

https://drive.google.com/file/d/1hSFt6FYg1XDCeqzg4VB1dv7iv8aXpybA/view?usp=sharing

**Comment by DImuthuUpe** [comment]
@kaulfusa Here is the outputs for hurricane Ida
https://drive.google.com/file/d/1-yEApTbL0K03tMRt819ihE3BC8R161xh/view?usp=sharing

**Comment by kaulfusa** [comment]
Are there comparison plots against the BM data that was already in VEDA for these cases?

**Comment by DImuthuUpe** [comment]
Yes. I uploaded those to the s3 bucket but it seems like the rendering workflow is broken. Probably because there are files with 2 different naming formats. @brianmfreitag might know the context

<img width="597" alt="Screenshot 2024-03-20 at 10 24 40 AM" src="https://github.com/NASA-IMPACT/veda-black-marble/assets/1346332/633ac3a0-7ce2-4ba5-b8b0-d5bdc9565874">
